go/vt/vttablet/tabletmanager/binlog_players.go

```diff
@@ -299,19 +300,24 @@ func (bpc *BinlogPlayerController) Iteration() (err error) {
 		return fmt.Errorf("not starting because flag '%v' is set", binlogplayer.BlpFlagDontStart)
 	}
 
+	sourceTabletType, err := topoproto.ParseTabletType(*sourceTabletTypeStr)
+	if err != nil {
+		log.Fatalf("unknown tablet type: %v", *sourceTabletTypeStr)
```

Comment:
Please use `return fmt.Errorf` e.g. as in a couple lines above.

We don't want to crash tablets when they're already running.